consensus/src/epoch_manager.rs

```diff
@@ -248,9 +254,20 @@ impl EpochManager {
                         + PROPSER_ROUND_BEHIND_STORAGE_BUFFER,
                     self.storage.aptos_db(),
                 ));
+                let voting_powers: Vec<_> = proposers
+                    .iter()
+                    .map(|p| {
+                        if weight_by_voting_power {
+                            epoch_state.verifier.get_voting_power(p).unwrap()
+                        } else {
+                            1
```

Comment:
Why default to 1 and not 0?

Reply:
this is not if validator doesn't exist (that should never be the case)
this is when configuraiton weight_by_voting_power=false is set, meaning we want previous behavior, which is all having equal weight of 1

Reply:
Ah I see. Can we make this a comment?

Reply:
nit: we should probably move the if condition out to make it more clear

Reply:
moved outside, I think it is clearer now, that comment is not needed